src/arch/arm/rte_atomic.h

```diff
@@ -7,7 +7,7 @@
 
 #if defined(__aarch64__)
 #include "arch/arm/rte_atomic_64.h"
-else
+#else
```

Comment:
why are we removing this?

Reply:
Not removing, just added `#` which was missing before the `else`.